source/fundamentals/collations.txt

```diff
@@ -4,5 +4,302 @@ Collations
 
 .. default-domain:: mongodb
 
-A collation is a set of character ordering and matching rules that
-apply to a specific language and locale.
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use **collations** to order your query
+or aggregation operation results by string values. A collation is a set of character
+ordering conventions that apply to a specific language and locale.
+
+Collations in MongoDB
+---------------------
+
+MongoDB sorts strings using *binary collation* by default. This collation
+method uses the `ASCII standard <https://en.wikipedia.org/wiki/ASCII>`_
+character values to compare and order strings. Certain languages and locales
+have specific character ordering conventions that differ from the ASCII
+standard.
+
+For example, in Canadian French, the right-most accented character determines
+the ordering for strings when the other characters are the same. Consider the
+following Canadian French words: 
+
+- cote
+- coté
+- côte
+- côté
+
+When using the default binary collation, MongoDB sorts them in the following order:
+
+.. code-block:: none
+
+   cote
+   coté
+   côte
+   côté
+
+When using the Canadian French collation, MongoDB sorts them in the following order:
+
+.. code-block:: none
+
+   cote
+   côte
+   coté
+   côté
+
+Specify a Collation
+-------------------
+
+To create a Collation object, you must specify a ``Locale`` field; all other fields are
```

Comment:
Issue: I believe "Collation" object was incorrectly introduced. I believe a reader must be made aware why "creating a Collation object" is necessary to achieve their goal.

Issue: I believe "Collation" in "Collation object" should be monospace as its a type name. This applies elsewhere in this paragraph.

```suggestion
You specify a collation through a ``Collation`` object.
To create a ``Collation`` object, you must specify a ``Locale`` field; all other fields are
```

Reply:
I included a sentence explaining that you create a collation object to specify a collation, and I used monospace for `Collation`.